server.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var pkg = require('./package.json');
+const shell = require('shelljs');
+var server = require('browser-sync').create(pkg.name);
+var options = require('command-line-args')([
+  { name: 'proxy',  alias: 'x', type: String },
+  { name: 'port',   alias: 'p', type: Number },
+  { name: 'static', alias: 'o', type: Boolean },
+  { name: 'tunnel', alias: 't', type: String },
+  { name: 'ssl',    alias: 's', type: Boolean },
+  { name: 'watch',    alias: 'w', type: Boolean }
+]);
+
+
+var proxy = options.proxy || 'bouncyball.dev';
+var port = options.port || 8080;
+var tunnel = options.tunnel || false;
+var https = options.ssl ? { key: './livereload.key', cert: './livereload.crt' } : false;
+
+let sharedSettings = {
+  port: port,
+  tunnel: tunnel,
+  https: https,
+  logPrefix: pkg.name,
+  open: false
+};
+
+// Configuration object passed to BrowserSync
+var config = {};
+
+// If we are using a proxy create a unique Object of settings
+if (options.proxy) {
+
+  config = Object.assign({
+    proxy: proxy
+  }, sharedSettings);
+
+} else {
+  // Rewrite URLs without .html to map to .html files
+  var rewriteModule = require('http-rewrite-middleware').getMiddleware([
+    { from: '(^((?!css|html|js|img|\/$).)*$)', to: '$1.html' }
+  ]);
```

Comment:
I'm getting an error on the `SMIL` example:

`GET http://localhost:8081/examples/smil/image.svg 404 (Not Found)`

I think it might have something to do with the filetypes specified in the code above. Here's a screenshot:

![screen shot 2016-09-30 at 6 18 34 pm](https://cloud.githubusercontent.com/assets/1256329/19008627/81a5e72e-873a-11e6-916c-234844d14fa4.png)